feature/011

JeremiasSanMartin opened the pull request:
Se creo el boton de LogOut en ActivityHome y se le dio la funcionalidad de borrar el nombre guardado en sharedpreference para luego devolver al usuario a Login
![image](https://github.com/Milton0123/Inmobiliaria/assets/94081789/778fa5e5-65a3-4293-940d-ba8f8b78d622)

JeremiasSanMartin committed:
feature/011

Se creo el boton de LogOut en ActivityHome y borra el nombre guardado en sharedpreference para luego devolver al usuario a Login

KevinMaggio commented on app/src/main/java/com/example/concesionaria/HomeActivity.kt:
<img width="510" alt="Captura de Pantalla 2023-08-02 a la(s) 17 39 25" src="https://github.com/Milton0123/Inmobiliaria/assets/86111731/bd28a381-fceb-426f-8c3c-52c5f8d505ee">
 la funcion se llama inicio. y no hace refencia al seteo de la acciones del boton. 
 podemos cambiarlo por 
 fun actions
 fun setOnClick

JeremiasSanMartin replied:
solucionado!

KevinMaggio commented on app/src/main/java/com/example/concesionaria/HomeActivity.kt:
<img width="500" alt="Captura de Pantalla 2023-08-02 a la(s) 17 42 03" src="https://github.com/Milton0123/Inmobiliaria/assets/86111731/bc8d33f8-ed48-4f24-990d-65ddc886e689">

esta funcion nos da la impresion que accede a un login. Pero no habla de moverse de pantalla ni nada. 

podemos usar como nombre
goToLogInView()

JeremiasSanMartin replied:
solucionado!

KevinMaggio commented on app/src/main/java/com/example/concesionaria/LoginActivity.kt:
veamos si encontramos el icono para la izquierda.  por q ese parece q es para entrar a algun lado.

JeremiasSanMartin replied:
solucionado!

JeremiasSanMartin committed:
code fix feature/011

Solucion a nombres de las funciones, tanto en login y en home para que sigan el mismo formato
icono rotado
